src/useInterval/useInterval.ts

```diff
@@ -32,5 +41,9 @@ export const useInterval = (
     if (startImmediate) start();
   });
 
-  return { startInterval: start, stopInterval: stop };
+  return {
+    startInterval: start,
+    stopInterval: stop,
+    isIntervalRunning: computed(() => isIntervalRunning.value),
```

Comment:
May I suggest:
```ts
isIntervalRunning: readonly(isIntervalRunning),
```

Reply:
Is this the preferred approach for these readonly / computed values? We do similar things on the component library where we have local refs but expose computed variants so they cannot be modified from the outside.

Reply:
I would think so, since it's not really computing anything. The `readonly` is made specifically for this purpose :)

Reply:
I'll implement that on the component library, because it does look pretty clean :)